### ISSUE TITLE
v2 Theme is not merging into current theme

### ISSUE THREAD

**Issue by atanasster** [grommet/grommet#2181]
When a theme is used to only change partially the theme behavior, it merges into the 'grommet' theme, instead of merging into the currently selected theme:

https://stackblitz.com/edit/grommet-demo-vfc2xf?file=index.js

Expected behavior is that the theme partials should be merged into the currently used theme

**Comment by ericsoderberghp** [comment]
I think we should document `ThemeContext.Extend` as the way to do this. It is already available, just not documented.

**Comment by alansouzati** [comment]
yup here is how to do it: https://stackblitz.com/edit/grommet-demo-2vbmwi?file=index.js

**Comment by ericsoderberghp** [comment]
We still need to document this in grommet-site
